src/pages/academy/grading/Grading.tsx

```diff
@@ -58,6 +59,10 @@ const Grading: React.FC = () => {
   const allColsSortStates = useTypedSelector(state => state.workspaces.grading.allColsSortStates);
   const hasLoadedBefore = useTypedSelector(state => state.workspaces.grading.hasLoadedBefore);
   const requestCounter = useTypedSelector(state => state.workspaces.grading.requestCounter);
+  const tokens = useTypedSelector(state => ({
+    accessToken: state.session.accessToken || 'Invalid',
+    refreshToken: state.session.refreshToken || 'Invalid'
+  }));
```

Comment:
I'm not a big fan of this, can we just do the following?

```suggestion
  const accessToken = useTypedSelector(state => state.session.accessToken);
  const refreshToken = useTypedSelector(state => state.session.refreshToken);
```